src/types.ts

```diff
@@ -89,6 +89,11 @@ export interface Instance {
     metadataUri: string,
     signer: ethers.Signer
   ): Promise<ethers.ContractTransaction>;
+  setDomainRoyalty(
```

Comment:
Please add a function header comment here to explain usage. Look at the other functions for the proper format

Reply:
Done 👍